smartsim/_core/_cli/scripts/dragon_install.py

```diff
@@ -182,7 +182,7 @@ def install_package(asset_dir: pathlib.Path) -> int:
         logger.info(f"Installing package: {wheel_path.absolute()}")
 
         try:
-            pip("install", "--force-reinstall", str(wheel_path))
+            pip("install", "--force-reinstall", str(wheel_path), "numpy<2")
```

Comment:
Actually one small note: It looks like the version of numpy that comes in when doing a "normal"
```
pip install --no-cache-dir smartsim[dev,mypy,ml] && smart build --onnx
```
is version 1.24.3 (at least for me, from pypi, on osx).

This change will presumably pin to [version 1.26.4](https://pypi.org/project/numpy/#history). I'm not sure if that matters, but if you want to be _SUPER_ safe, you may want to pin to that number.